cmd/vaRequirement.go

```diff
@@ -49,19 +83,76 @@ var vaRequirementUpdateCmd = &cobra.Command{
 			$ vf-admin va requirement update c7bc794c-9905-4588-81e6-e557e1a494c4 a9620b24-0dc4-4c7d-8ca2-a9b06d627d82 --requirement 10 --active=false
 	`),
 	Args: cobra.ExactArgs(2),
-	RunE: update.CmdRunE,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		var op update.HTTPOperation
+
+		// Retrieve the authentication key from configuration file
+		key, kErr := utils.GetKeyFromProfile(cmd)
+		if kErr != nil {
+			color.Red(kErr.Error())
+			return nil
+		}
+
+		if err := op.SetRequestURLArguments(args); err != nil {
+			return err
+		}
+
+		flags := cmd.Flags()
+
+		var requirement *int
```

Comment:
why are there pointers?

Reply:
There's no use in having pointers and my IDE is giving me warnings. I think you should look more closely at some other modules as examples, the pointer was used for other reasons.